tasks-app-shared/src/iosMain/kotlin/net/opatry/tasks/app/ui/component/AuthorizeGoogleTasksButton.ios.kt

```
@@ -0,0 +1,100 @@
+                coroutineScope.launch {
+                    val scope = listOf(
+                        GoogleAuthenticator.Scope.Profile,
+                        GoogleAuthenticator.Scope(TasksScopes.Tasks),
+                    )
+                    try {
+                        val authCode = authenticator.authorize(scope, true) {
+                            // TODO deal with iOS
+                        }.let(GoogleAuthenticator.Grant::AuthorizationCode)
+                        val oauthToken = authenticator.getToken(authCode)
+                        onSuccess(oauthToken)
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)